tests/php/Core/ClassInfoTest.php

```diff
@@ -87,6 +87,14 @@ public function testSubclassesFor()
             $subclassesWithoutBase,
             ClassInfo::subclassesFor('silverstripe\\core\\tests\\classinfotest\\baseclass', false)
         );
+
+        // Check that core classes are present (eg: Email subclasses)
+        $emailClasses = ClassInfo::subclassesFor(SilverStripe\Control\Email\Email::class);
```

Comment:
Try this.

```suggestion
        $emailClasses = ClassInfo::subclassesFor(\SilverStripe\Control\Email\Email::class);
```

Reply:
@maxime-rainville now i feel stupid to have missed that :-) done !